docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -559,9 +555,18 @@ Treatment Effects
 
 **Blood Pressure Treatments**  
 
+.. todo::
+  - Add parameter variation if needed to SBP  
+
 Blood pressure treatment efficacy is dependent on a simulant's SBP value. Full efficacy data is here:
 /share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_sbp_new.csv [Law_2009]_
 
+For each seed/draw, a parameter value for efficacy will be selected based on table above. While we plan 
```

Comment:
I think we typically just vary parameter uncertainty just at the draw level, not draw/seed level.